appium-dotnet-driver/Appium/Tizen/TizenDriver.cs

```diff
@@ -82,6 +85,29 @@ public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabi
         {
         }
 
+        public bool CompareToImages(Image origianalImage, Image compareImage)
```

Comment:
API signature looks wired for me. 
What about `CompareImages(Image sourceImage, Image targetImage)` instead?